Sprout/View Controllers/PostDetailsViewController.m

```diff
@@ -31,23 +32,76 @@ - (void)viewDidLoad {
 }
 
 - (void)refreshData {
-    PFUser *currUser = self.post.author;
-    PFFileObject *photo = currUser[@"profileImage"];
+    PFUser *postUser = self.post.author;
+    PFFileObject *photo = postUser[@"profileImage"];
     [photo getDataInBackgroundWithBlock:^(NSData * _Nullable imageData, NSError * _Nullable error) {
         self.profileImageView.image = [UIImage imageWithData:imageData];
     }];
     self.timestampLabel.text = self.post.createdAt.shortTimeAgoSinceNow;
-    if([currUser[@"name"] isEqual:nil]){
+    if([postUser[@"name"] isEqual:nil]){
         self.usernameLabel.text = nil;
-        self.nameLabel.text = currUser.username;
+        self.nameLabel.text = postUser.username;
     }
     else{
-        self.usernameLabel.text = currUser.username;
-        self.nameLabel.text = currUser[@"name"];
+        self.usernameLabel.text = postUser.username;
+        self.nameLabel.text = postUser[@"name"];
     }
     self.progressLabel.text = [NSString stringWithFormat:@"I completed %@ of %@ tasks today!", self.post.completedTasks, self.post.totalTasks];
     self.captionLabel.text = self.post.caption;
     self.timestampLabel.text = self.post.createdAt.shortTimeAgoSinceNow;
+    self.likeCountLabel.text = [NSString stringWithFormat:@"%d likes", [self.post.likeCount intValue]];
+    
+    NSLog(@"%@", self.post.usersWhoLiked);
+    if([self.post.usersWhoLiked containsObject:PFUser.currentUser[@"email"]])
+    {
```

Comment:
This formatting is funny but it's consistent and that's more important for formatting. Can get used to reading anything if it's consistent. Typically it's `if (somecondition) {` all on the same line.